features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml

```diff
@@ -1670,4 +1670,17 @@
                    description="Delete workflow association"/>
         </Scopes>
     </APIResource>
+    <APIResource name="Workflow Approval Task API" identifier="me/approval-tasks" requiredAuthorization="true"
+                 description="API representation of the workflow approval Task API" type="TENANT">
+        <Scopes>
+            <Scope displayName="View Workflow Approval Task" name="internal_approval_task_view"
+                   description="View workflow approvals Tasks"/>
+            <Scope displayName="Create Workflow Approval Task" name="internal_approval_task_create"
+                   description="Create new workflow approval Tasks"/>
+            <Scope displayName="Update Workflow Approval Task" name="internal_approval_task_update"
+                   description="Update workflow approval Tasks"/>
+            <Scope displayName="Delete Workflow Approval Task" name="internal_approval_task_delete"
+                   description="Delete workflow approval Tasks"/>
```

Comment:
Let's have only view and update scopes.

We can't create or delete approval tasks.